### ISSUE TITLE
Issues streaming h.265 streams on Stan & Binge (stuttering)

### ISSUE THREAD

**Issue by camokarzi** [matthuisman/slyguy.addons#96]
Hi. First of all thankyou for all the work you do with the apps for kodi. Big fan !!

I have been using many different hardware types for kodi RPI3 & 4, Android etc
I have constant issues streaming Stan & Binge H.265 movies no matter the bitrate reduction I choose

Problem: movies pause , audio bitrate info goes blank intermittently then resumes on H.265 codec streams

My hardware is the T95 Xgody 4 x A53 cpu with Mali GPU runing android 10
RPI3 & 4  libreelec works flawlessly  ...love it
T95 has constant issues streaming with matrix 19.3 or liea 18.9
Streaming H.265 from USB  perfect
Streaming H.265 over local network perfect
Streaming h.265 via jellyfin server over internet perfect.
I suspect an issue with the way adaptivestream works but a little unsure. 

Could you please help me understand why this might be going wrong, spent many hours trying to rack my head and 100s of different setting changes make no difference.

TIA

**Comment by matthuisman** [comment]
try Kodi 20 with latest inputstream adaptive.
you should fine it works with H265 better

**Comment by camokarzi** [comment]
Will do, Thanks for the fast reply.

**Comment by matthuisman** [comment]
report back here on if it helps or not

**Comment by camokarzi** [comment]
Okay just installed Kodi 20 Alpha and deleted/ reinstalled inputstream   result is exactly the same unfortunately

**Comment by matthuisman** [comment]
really not sure then sorry

**Comment by camokarzi** [comment]
No worries I will keep at it and if I find a fix  will report back

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by camokarzi** [comment]
One last question ...sorry.. I noticed there is a 20.2 release on github but Kodi 20 only has the 19.0 install package.  how can in install the latest 20.2 package from github ?
________________________________
From: Paul Musumeci ***@***.***>
Sent: Tuesday, 7 December 2021 10:18 AM
To: matthuisman/slyguy.addons ***@***.***>; matthuisman/slyguy.addons ***@***.***>
Cc: Author ***@***.***>
Subject: RE: [matthuisman/slyguy.addons] Issues streaming h.265 streams on Stan & Binge (stuttering) (Issue #96)


No worries I will keep at it and if I find a fix  will report back



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

**Comment by matthuisman** [comment]
oh yeh, you've gotta use the 20 version of Inputstream Adapative. haha.
no wonder didnt work.
I assume Kodi 20 would use the 20 IA by default

**Comment by camokarzi** [comment]
Yep  Kodi V20 only gives 19.0 IA  anyway I can grab the package as a ready made zip ?
________________________________

**Comment by matthuisman** [comment]
You can actually install the IA 20 version on Kodi 19 as well.

your Android?

**Comment by camokarzi** [comment]
yep android 10 on t95 xgody mbox
________________________________

**Comment by matthuisman** [comment]
https://jenkins.kodi.tv/job/xbmc/job/inputstream.adaptive/job/Nexus/34/artifact/cmake/addons/build/zips/inputstream.adaptive+android-armv7/inputstream.adaptive-20.0.2.zip

**Comment by camokarzi** [comment]
Thanks for that installed but same result...bugger.
________________________________

**Comment by matthuisman** [comment]
same even if you reduce quality? eg. try the lowest H265 stream

**Comment by camokarzi** [comment]
Yes try the lowest quality  however still stutters and stops within 3 secs



Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
